Modules/BenchPress.Azure/Tests/Public/Get-ResourceByType.Tests.ps1

```diff
@@ -1,34 +1,72 @@
-﻿BeforeAll {
-  . $PSScriptRoot/../../Public/Confirm-AppServicePlan.ps1
-  . $PSScriptRoot/../../Public/Confirm-ResourceGroup.ps1
-  . $PSScriptRoot/../../Public/Confirm-SqlDatabase.ps1
-  . $PSScriptRoot/../../Public/Confirm-SqlServer.ps1
-  . $PSScriptRoot/../../Public/Confirm-VirtualMachine.ps1
-  . $PSScriptRoot/../../Public/Confirm-WebApp.ps1
+﻿using module ./../../Classes/ResourceType.psm1
+
+BeforeDiscovery {
+  $testCases = @()
+
+  foreach ($i in [ResourceType].GetEnumNames())
```

Comment:
Can you change this to something more descriptive than `$i`? The obvious one is `$resourceType`